### ISSUE TITLE
non-ASCII characters

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1631]
I see the following on my work machine. (I'll check later on my home machine, but I don't like getting a NOTE on *either* machine because a NOTE means CRAN might refuse the submission.)

I can't look at this for a while.  The point of this issue is so I can find the solution later. This seems to come up every few months, and I don't remember the thought process for getting rid of it.

Somewhere, we need to tell R that a particular file has at least one non-ASCII character. Likely it's a "theta" in a cnv file, but it could be anything, e.g. a header might have a person or place name that has an accent.  Why R complains about departure from a convention that we got past 50 years ago, I do not know :-)

```R
* checking data for non-ASCII characters ... NOTE
  Note: found 1 marked UTF-8 string
```

**Comment by dankelley** [comment]
This find them:
```R
> library(tools) # for showNonASCIIfile()
> for(file in list.files()) {bad<-showNonASCIIfile(file);if (!is.null(bad)) cat(bad)}
154: \code{sigma-<c3><a9>}     \tab \code{sigmaTheta}                   \tab kg/m^3               \tab 5 \cr
\code{sigma-é}     \tab \code{sigmaTheta}                   \tab kg/m^3               \tab 5 \cr127: 21, no. 4 (April 1, 2004): 704<e2><80><93>15.
21, no. 4 (April 1, 2004): 704–15.Error in file(con, "r") : cannot open the connection
```

**Comment by dankelley** [comment]
The second is from a copy/paste from Zotero: the dash in the page range is not an ascii hyphen but rather something else.  Fixed in "sf" branch commit bc76e48d9b1bb7e9eb1bd925938ef678c119976a (I should have fixed in the "develop" branch but I didn't notice what branch I was on.  Anyway, "sf" is likely to get merged into "develop" this weekend.)

**Comment by dankelley** [comment]
This is fixed in the `sf` branch.  I am closing the issue, even though the fix is not in the `develop` branch, because I am the reporter.
